gulpfile.js

```diff
@@ -115,7 +115,7 @@ gulp.task('scripts', ['lint'], function scriptsTask() {
   return bundledStream;
 });
 
-gulp.task('styles', function stylesTask() {
+gulp.task('styles', ['sass-lint'], function stylesTask() {
```

Comment:
Hmm, this seems to [just work](https://travis-ci.org/mozilla/idea-town/jobs/91921433#L585) in Travis and my dev machine. I expected it to break, because I've never installed the scss_lint ruby gem locally and .travis.yml doesn't do it either. Cool that it's working, but not sure why :)